File: Code/Python/LAB06_02.py
def main():
    # สร้าง list สำหรับเก็บคะแนน
    scores = []

    # รับคะแนนของนักเรียน 10 คน
    print("Enter the scores for 10 students:")
    for i in range(1, 11):
        score = int(input(f"Student {i}: "))
        scores.append(score)

    # คำนวณผลรวมคะแนน
    total_score = sum(scores)

    # คำนวณค่าเฉลี่ย
    average_score = total_score / len(scores)

    # แสดงผลคะแนนของนักเรียนแต่ละคน
    print("\nScores of Students:")
    for i, score in enumerate(scores, start=1):
        print(f"Student {i}: {score}")

    # แสดงผลรวมคะแนนและค่าเฉลี่ย
    print(f"\nTotal Score: {total_score}")
    print(f"Average Score: {average_score:.2f}")

if __name__ == "__main__":
    main()

File: Code/Cplus/LAB06_02/LAB06_02.cpp
#include <iostream>
using namespace std;

int main() {
    // สร้าง Array สำหรับเก็บคะแนน
    int scores[10];
    int totalScore = 0;
    double averageScore;

    // รับคะแนนของนักเรียน 10 คน
    cout << "Enter the scores for 10 students:" << endl;
    for (int i = 0; i < 10; i++) {
        cout << "Student " << i + 1 << ": ";
        cin >> scores[i];
        totalScore += scores[i]; // เพิ่มคะแนนเข้าสู่ผลรวม
    }

    // คำนวณค่าเฉลี่ย
    averageScore = static_cast<double>(totalScore) / 10;

    // แสดงผลคะแนนของนักเรียนแต่ละคน
    cout << "\nScores of Students:" << endl;
    for (int i = 0; i < 10; i++) {
        cout << "Student " << i + 1 << ": " << scores[i] << endl;
    }

    // แสดงผลรวมคะแนนและค่าเฉลี่ย
    cout << "\nTotal Score: " << totalScore << endl;
    cout << "Average Score: " << averageScore << endl;

    return 0;
}
